spec/models/dashboard_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+require "rails_helper"
```

Comment:
ah, that skipped through the read. Is it worth the tradeoff? Probably it is only a matter of time until we have to switch it anyway.